silicon/static/css/colors-light.css

```diff
@@ -1,27 +1,67 @@
 :root {
-    --nav-background: rgb(221, 221, 221);
-    --nav-text: black;
-    --nav-title-background: rgb(52, 58, 64);
-    --nav-title-text: white;
-    --nav-page-links: rgb(51, 122, 183);
-    --nav-text: rgb(119, 119, 119);
-    --nav-button-background: var(--nav-page-links);
-    --nav-button-background-hover: rgba(51 ,122 ,183 ,0.75);
-    --nav-button-text: white;
-    --nav-alert-background: rgb(183, 52, 52);
-    --nav-alert-text: white;
-
-    --article-background: white;
-    --article-code-text: black;
-    --article-pre-background: rgb(236, 237, 239);
-    --article-pre-text: black;
-    --article-pre-border: rgb(214, 216, 219);
-    --article-table-border: var(--article-pre-border);
-    --article-table-stripe: var(--article-pre-background);
+    /* Color palette */
+    --c-gray10: white;
+    --c-gray9: rgb(236, 237, 239);
+    --c-gray8: rgb(221, 221, 221);
+    --c-gray7: rgb(214, 216, 219);
+    --c-gray5: rgb(128, 128, 128);
+    --c-gray4: rgb(119, 119, 119);
+    --c-gray2: rgb(52, 58, 64);
+    --c-gray0: black;
```

Comment:
The numbers are are not deeply considered, I just thought it made sense to pin 128/128/128 as "middle gray" and spread everything out somewhat evenly.

Reply:
Seems like a good approach. I tried to be fairly careful with how I picked the grays. It's tough because every display has a different baseline and contrast.

Reply:
Conventional designer wisdom is to use fewer shades of similar grays and almost never use zero-saturation gray, but that's a topic for another time.